checkov/terraform/checks/resource/gcp/GoogleStorageBucketNotPublic.py

```diff
@@ -0,0 +1,23 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class GoogleStorageBucketNotPublic(BaseResourceCheck):
+    def __init__(self):
+        name = " Ensure that Cloud Storage bucket is not anonymously or publicly accessible"
+        id = "CKV_GCP_28"
+        supported_resources = ['google_storage_bucket_iam_member', 'google_storage_bucket_iam_binding']
+        categories = [CheckCategories.IAM]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if 'member' in conf.keys():
+            member_conf = conf['member']
+        elif 'members' in conf.keys():
+            member_conf = conf['members'][0]
```

Comment:
@arielkru To simplify these 4 lines, I would write it using the `get` method, which allows for optional keys. i.e.:
`member_conf = conf.get('member', []) + conf.get('members', [[]])[0]`